Rock/Model/CodeGenerated/ReportService.cs

```diff
@@ -53,6 +54,43 @@ public bool CanDelete( Report item, out string errorMessage )
             errorMessage = string.Empty;
             return true;
         }
+
+        /// <summary>
+        /// Create a new non-persisted Report using an existing Report as a template. 
+        /// </summary>
+        /// <param name="reportId">The identifier of a Report to use as a template for the new Report.</param>
+        /// <returns></returns>
+        public Report GetNewFromTemplate( int reportId )
```

Comment:
@azturner Sorry, I should have been alert to the fact that it was in a folder named "CodeGenerated"! Thanks for picking this up.